demo/demo/tests/test_trial_period.py

```diff
@@ -0,0 +1,41 @@
+from subscriptions.api.views import SubscriptionSelectView
```

Comment:
please add a test that shows that I wouldn't be able as a user setup a cron job to cancel & start new free trial everyday